src/plugins/discover/public/application/components/doc_viewer/doc_viewer.scss

```diff
@@ -45,7 +45,6 @@
 }
 
 .kbnDocViewer__value {
-  display: inline-block;
```

Comment:
It’s just a small finding which improves doc viewer table row alignment (now really centered vertically). It was not done in corresponding doc viewer improvement PR before. See difference in the picture. @andreadelrio could you please take a look at it?

 Before `display: inline-block` removal

![EA71EB6C-32D0-4CB9-AE54-14F3B802B06D_4_5005_c](https://user-images.githubusercontent.com/39378793/126131432-89e2a558-b0e5-4c42-aad2-fe25582a7507.jpeg)

After `display: inline-block` removal

![2AE50B29-DAB9-440C-BC12-DDA01C4AE3AA_4_5005_c](https://user-images.githubusercontent.com/39378793/126131646-98c594c7-cbb9-491a-8060-9b2ce1a818e7.jpeg)

Reply:
While this visually fixes the issue, I'm not certain this `display` rule is the culprit here. I did some tinkering and noticed that there seems to be an extra div wrapper that, when removed, also seems to fix the issue.

Can the plain `<div>` around the`kbnDocViewer__value` element be removed altogether?

<img width="293" alt="Screen Shot 2021-08-04 at 9 23 42 AM" src="https://user-images.githubusercontent.com/446285/128198514-b95f53f7-7199-4b13-8d85-88d3e2056bb5.png">

Reply:
Yes, thank you. I have created [separate PR](https://github.com/elastic/kibana/pull/107869), which you can check. Since I have some other adjustings in doc-viewer.